app/src/main/res/layout/fragment_main.xml

```diff
@@ -61,7 +59,6 @@
             android:layout_weight="0.2"
             android:layout_height="match_parent"
             android:contentDescription="@string/nav_item_more"
-            android:background="?attr/selectableItemBackground"
```

Comment:
What's the reason for removing this?  The "More" item no longer has a ripple effect when tapping it.